src/gridstack.ts

```diff
@@ -1677,17 +1621,27 @@ export class GridStack {
     return this;
   }
 
-  /** @internal call to write position x,y,w,h attributes back to element */
-  protected _writePosAttr(el: HTMLElement, n: GridStackPosition): GridStack {
+  /**
+   * Call to write position x,y,w,h attributes back to element
+   * In addition, updates the inline top/height inline style as well
+   * @internal
+   */
+  protected _writePosAttr(el: HTMLElement, n: GridStackNode): GridStack {
     if (n.x !== undefined && n.x !== null) { el.setAttribute('gs-x', String(n.x)); }
     if (n.y !== undefined && n.y !== null) { el.setAttribute('gs-y', String(n.y)); }
     n.w > 1 ? el.setAttribute('gs-w', String(n.w)) : el.removeAttribute('gs-w');
     n.h > 1 ? el.setAttribute('gs-h', String(n.h)) : el.removeAttribute('gs-h');
+    // Avoid overwriting the inline style of the element during drag/resize, but always update the placeholder
+    if ((!n._moving && !n._resizing) || this._placeholder === el) {
+      // Set inline style, refer CSS variables
+      el.style.top = `calc(${n.y} * var(--gs-cell-height))`;
+      el.style.height = `calc(${n.h} * var(--gs-cell-height))`;

```

Comment:
remove height style when =1 (set globally once) - lot less rules when you have a ton of 1x1 elements

Reply:
Having the rule always set, even in 1x1 case, is IMO more readable and more robust to future refactoring. However, pushed the required change.